### DISCUSSION TITLE
How to get a track with timestamps?

### DISCUSSION THREAD

**Discussion by printmaps** [devemux86/cruiser#658] [Q&A] [answered]
I share recorded tracks via eMail. But these tracks have no timestamps (see screenshot from RouteConverter).

<img width="1407" alt="Bildschirmfoto 2024-10-27 um 07 13 55" src="https://github.com/user-attachments/assets/5a17d176-26a0-4b6e-bc90-53d62a6c751e">

In another thread this was mentioned (from emux):
![380372731-658106bc-f342-4121-9610-33a01dfb79c0](https://github.com/user-attachments/assets/6923f75e-47cd-4ac9-94fd-2738bbcfe673)

Question: What to configure in order to get timestamps in a saved track?

**Comment by devemux86** [comment]
@printmaps The answer is already above.

Recorded tracks have all GPS provided data: coordinates, elevation, time, accuracy.

When you export the recorded track with the menu "GPS recording | Export" or by pressing the GPS recording panel,
you can select the data you want to be included in the exported gpx file, like in the above image.

**Reply by devemux86** [reply]
Example of an exported recorded gpx: [recorded.gpx.txt](https://github.com/user-attachments/files/17533220/recorded.gpx.txt)

**Comment by printmaps** [comment]
Clarification:

I want to use the Cruiser internal track management (see screenshot). In other words, I want to save the track only (no additional export).

![Screenshot_20241027_092531_Cruiser](https://github.com/user-attachments/assets/893551ea-9706-4ba1-9325-909374dd5ff4)

Problem: If I share a track from the internal track management (e.g. by sending the track via eMail), I don't see any timestamps.

**Reply by devemux86** [reply, marked as answer]
Tracks in My collection currently do not store the time, just the geometry and elevation.
As it was not in the requirements discussed with users when we created the favorite tracks.

I can see it, if it is needed by the users.

**Comment by printmaps** [comment]
For me the timestamps are essential. Without them a lot of informations (begin, end, speed, no movement, ...) are missing.

**Reply by devemux86** [reply]
@printmaps Please create a new thread with your proposal so that it does not get lost in the discussion here.

**Comment by printmaps** [comment]
Done.
